ethpm_cli/registry.py

```diff
@@ -25,8 +26,34 @@ def format_for_display(self) -> str:
         return f"{self.uri} --- {self.alias}{activated}"
 
 
+def deploy_registry(config: Config, alias: str = None) -> str:
+    if not config.private_key:
+        raise AuthorizationError(
+            "To deploy a registry, you must provide the password for your local keyfile."
+        )
+    chain_id = config.w3.eth.chainId
+    chain_name = SUPPORTED_CHAIN_IDS[chain_id]
+    cli_logger.info(
+        f"Deploying a new registry to {chain_name}, this may take a minute..."
+    )
+    # todo: handle tx timeout error gracefully
+    registry_address = config.w3.pm.deploy_and_set_registry()
+    cli_logger.info(f"New registry deployed to {registry_address} on {chain_name}.")
+    registry_uri = URI(f"erc1319://{registry_address}:{chain_id}")
+    add_registry(registry_uri, alias, config)
+    activate_registry(registry_uri, config)
+    explorer_uri = f"http://explorer.ethpm.com/browse/{chain_name}/{registry_address}"
```

Comment:
I don't think it's right for this function to return the explorer URI.  I'd advocate for having it return the address and to do the URI construction elsewhere.